app/lib/actions/katello/repository/clone_docker_content.rb

```diff
@@ -4,7 +4,7 @@ module Repository
       class CloneDockerContent < Actions::Base
         def plan(source_repo, target_repo)
           sequence do
-            plan_action(Pulp::Repository::CopyDockerImage,
+            plan_action(Pulp::Repository::CopyDockerManifest,
```

Comment:
@beav, good question!  They are.  I did check the output of the target repo after the copies completed to ensure that the counts for blob/manifest/tag match that of the source repo.